demisto_sdk/commands/common/hook_validations/docker.py

```diff
@@ -51,17 +51,10 @@ def is_docker_image_latest_tag(self):
             self.is_latest_tag = False
             return self.is_latest_tag
 
-        server_version = LooseVersion(self.from_version)
-        # Case of a modified file with version >= 5.0.0
-        if self.is_modified_file and server_version >= '5.0.0':
-            if self.docker_image_latest_tag != self.docker_image_tag and not \
-                    'demisto/python:1.3-alpine' == '{}:{}'.format(self.docker_image_name, self.docker_image_tag):
-                # If docker image name are different and if the docker image isn't the default one
-                self.is_latest_tag = False
-        # Case of an added file
-        elif not self.is_modified_file:
-            if self.docker_image_latest_tag != self.docker_image_tag:
-                self.is_latest_tag = False
+        if self.docker_image_latest_tag != self.docker_image_tag and not \
+                'demisto/python:1.3-alpine' == '{}:{}'.format(self.docker_image_name, self.docker_image_tag):
```

Comment:
we should allow using `demisto/python:1.3-alpine`

Reply:
It is allowing there is an "and not' before the statement, I think I'll separate it to 2 statement so it will be more readable